src/workspaces/list/NoContentMessage.tsx

```diff
@@ -17,15 +15,11 @@ export const NoContentMessage = (props: NoContentMessageProps): ReactNode => {
   const { loadingWorkspaces, workspaces, filters } = props;
   const { setUserActions } = useContext(WorkspaceUserActionsContext);
 
-  return cond(
-    [loadingWorkspaces, () => h(Fragment, ['Loading...'])],
-    [
-      _.isEmpty(workspaces.myWorkspaces) && filters.tab === 'myWorkspaces',
-      () =>
-        NoWorkspacesMessage({
-          onClick: () => setUserActions({ creatingNewWorkspace: true }),
-        }),
-    ],
-    () => div({ style: { fontStyle: 'italic' } }, ['No matching workspaces'])
-  );
+  if (loadingWorkspaces) {
+    return <> Loading... </>;
+  }
+  if (_.isEmpty(workspaces.myWorkspaces) && filters.tab === 'myWorkspaces') {
```

Comment:
This isn't showing up here because the file was modified, but I noted that you might prefer to use some combination of the conditonal operator (?.) and defaults (??) here. Something like: `workspaces?.myWorkspaces?.length ?? 0`.
Nothing wrong with it as is,though.